doc/docs.md

```diff
@@ -2927,14 +2927,29 @@ The developer doesn't need to change anything in their code. "It just works", li
 Python, Go, or Java, except there's no heavy GC tracing everything or expensive RC for
 each object.
 
+### Control
+
+You can take advantage of V's autofree engine and define a `free()` method on custom 
+data types:
+
+```v
```

Comment:
You need to either define `struct MyType {}`, or use `v oksyntax` to avoid checking.